src/main/kotlin/sliv/tool/model/project/ProjectFrame.kt

```diff
@@ -0,0 +1,7 @@
+package sliv.tool.model.project
+
+import java.nio.file.Path
+import java.time.LocalDateTime
+
+data class ProjectFrame(val timestamp: LocalDateTime, val imagePath: Path, val outputs: List<AlgorithmOutput>) {
```

Comment:
timestamp is long, why real date?

Reply:
how do you usually store timestamps as names for frames?